resttools/models/irws.py

```diff
@@ -136,6 +136,7 @@ class SdbPerson(object):
     fname = ''
     lname = ''
     categories = ()
+    contact_email = ()
```

Comment:
I don't know the difference between '' and ( )

Reply:
I believe it's intended to imply that it can be a series of values, vs. a single value. Unfortunately this package is used by some outdated stuff so can't use type annotations.